pkg/console/operator/sync_v400.go

```diff
@@ -593,3 +601,14 @@ func (co *consoleOperator) GetAvailablePlugins(enabledPluginsNames []string) []*
 	}
 	return availablePlugins
 }
+
+func getSupportedArchitectures(nodes *corev1.NodeList) []string {
```

Comment:
Is this going to trigger unnecessary rollouts if the order of the nodes changes (since the order in the arch potentially changes too)? When does this get evaluated and how we decide when to rollout a new console version?

Reply:
@spadgett good catch. Updated the PR to address the node informer and also sort the `nodesArch` array so it wont get triggered if the order changes.